src/chrome/content/scaffold/templates/functions.txt

```diff
@@ -0,0 +1,47 @@
+These special Zotero functions are most frequent used in translator coding, for autocompletion try also ctrl+space during writing.
```

Comment:
`These are the most common functions used in Zotero translators.`

Does the second part only apply if the live option is off?